user/languages/r.md

```diff
@@ -256,6 +256,9 @@ processed in order, so entries can depend on dependencies in a previous list.
   using `devtools::install_github` from the
   [devtools package][github 8]. The package names
   here should be of the form `user/repo`.
+  If the package is installed in a folder of the GitHub, use `user/repo/folder`.
+  An alternative is to add `user/repo` or `user/repo/folder` to
+  the `Remotes` section of the `DESCRIPTION` file of your package
```

Comment:
I'm not very keen on "in a folder of the GitHub". Maybe just "if the package is installed in a subdirectory use `user/repo/subdirectory`?

And maybe provide an example of the Remotes use?